setup.py

```diff
@@ -282,6 +282,7 @@ def find_py(self):
             "rhsm-facts-service = subscription_manager.scripts.rhsm_facts_service:main",
             "rhsm-service = subscription_manager.scripts.rhsm_service:main",
             "rhsmcertd-worker = subscription_manager.scripts.rhsmcertd_worker:main",
+            "package-profile-uploader = subscription_manager.scripts.package_profile_upload:main",
```

Comment:
since this new script/tool will be installed in LIBEXEC (`/usr/libexec` on Red Hat systems) and that directory is shared among all the other system packages, then we still cannot use too generic names (there's the potential risk of name clashes)

please add a `rhsm-` prefix like for the other scripts